app/Http/Controllers/AuditLogController.php

```diff
@@ -16,4 +16,42 @@ public function table(DataTables $dataTables)
     {
         return $dataTables->queryBuilder(DB::table('audit_logs'))->make(true);
     }
+
+    public function delete(){
+        $result = array();
+        $deletedRows = DB::table('audit_logs')->delete();
+
+        if ($deletedRows > 0) {
```

Comment:
Remove if else.... just return message success and just run DB::table('audit_logs')->delete();